packages/itwinui-css/backstop/tests/button.html

```diff
@@ -241,35 +290,53 @@ <h3>Call to action</h3>
         id="demo-cta-small"
         class="demo-container"
       >
-        <button class="iui-button iui-cta iui-small">
+        <button
+          class="iui-button"
+          data-iui-button-variant="cta"
+          data-iui-button-size="small"
+        >
           <span class="iui-button-label">Small</span>
```

Comment:
By the way, you have renamed this in header, should we align???

Reply:
In #640 I am using `class="iui-header-breadcrumb-button"` seen [here](https://github.com/iTwin/iTwinUI/blob/a8eb0e12ff8f1703b2867fdb158af74283508c83/packages/itwinui-css/backstop/tests/header.html#L122).  Are you suggesting something like `<button class="iui-button" data-iui-button-variant="header-breadcrumb">`?

Reply:
In header we have 
``` 
text
  label
  sublabel
```

In Button we have just `label`. So we might want to align the naming, at least

Reply:
Replaced `.iui-button-label` with `.iui-button-text`.